### DISCUSSION TITLE
Microsoft 365 community feedback -

### DISCUSSION THREAD

**Discussion by Humra87** [MicrosoftDocs/microsoft-365-community#979] [General]
### Type of issue

Other (describe below)

### Feedback

Permissions on SharePoint - unsure if either Team or Communications site works for what we would like.

Can we amend permissions within the folders and/or files within either Teams or a Communications site. Basically we want to create a HR system where we can share content, a bit like a Teams site, but view only for some, but edit for others. It would be a site that has personnel and induction files for each staff/freelance member, so some people are outside our organisation.

We would like to, within their team files allow them to view certain documents (such as DBS, Training), but also want them to update parts of it (Uploading a training certificate, updating their DBS or PLI, signing forms regarding safeguarding etc.)

Is this something the Communication site could do? Or would Teams be better?

Thank you
Humira

### Page URL

https://learn.microsoft.com/en-us/microsoft-365/community/team-site-or-communication-site

### Content source URL

https://github.com/MicrosoftDocs/microsoft-365-community/blob/main/Community/team-site-or-communication-site.md

### Author

@SusanHanley

### Document Id

fe8166b4-1ba5-6312-f52d-acc365de2eee

**Comment by SusanHanley** [comment]
A real HRIS would be preferable because the permissions will be complicated to maintain in SharePoint and an HRIS is truly designed for this type of secure interaction, but you could technically do this using either type of site template.

From: Humra87 ***@***.***>
Sent: Friday, August 2, 2024 11:33 AM
To: MicrosoftDocs/microsoft-365-community ***@***.***>
Cc: Susan Hanley ***@***.***>; Mention ***@***.***>
Subject: [MicrosoftDocs/microsoft-365-community] Microsoft 365 community feedback - (Issue #967)

Type of issue

Other (describe below)

Feedback

Permissions on SharePoint - unsure if either Team or Communications site works for what we would like.

Can we amend permissions within the folders and/or files within either Teams or a Communications site. Basically we want to create a HR system where we can share content, a bit like a Teams site, but view only for some, but edit for others. It would be a site that has personnel and induction files for each staff/freelance member, so some people are outside our organisation.

We would like to, within their team files allow them to view certain documents (such as DBS, Training), but also want them to update parts of it (Uploading a training certificate, updating their DBS or PLI, signing forms regarding safeguarding etc.)

Is this something the Communication site could do? Or would Teams be better?

Thank you
Humira

Page URL

https://learn.microsoft.com/en-us/microsoft-365/community/team-site-or-communication-site

Content source URL

https://github.com/MicrosoftDocs/microsoft-365-community/blob/main/Community/team-site-or-communication-site.md

Author

@SusanHanley<https://github.com/SusanHanley>

Document Id

fe8166b4-1ba5-6312-f52d-acc365de2eee
